src/main/scala/com/github/unchama/seichiassist/subsystems/shareinventory/ShareInventoryAPI.scala

```diff
@@ -0,0 +1,33 @@
+package com.github.unchama.seichiassist.subsystems.shareinventory
+
+import com.github.unchama.seichiassist.subsystems.shareinventory.domain.bukkit.InventoryContents
+
+import java.util.UUID
+
+trait ShareInventoryWriteAPI[F[_]] {
```

Comment:
```suggestion
trait SharedInventoryWriteAPI[F[_]] {
```

「共有インベントリ」なので、「共有された」→ `Shared...`のほうが適切です。以下同様。